models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from uuid import uuid4
+
+class Garment(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    name = models.CharField(max_length=200)
+    dscription = models.TextField(blank=True)
```

Comment:
This is a possible breakpoint in the future, but if you're mindful that the property name is `dscription`, then this is not a big deal. Otherwise, might be good to fix this:

```suggestion
    description = models.TextField(blank=True)
```